besmarts-core/python/examples/07_besmarts_fit.py

```diff
@@ -140,11 +140,6 @@ def new_gdb(f: Dict[str, List[Dict[int, str]]]) -> assignments.graph_db:
         g = gcd.smiles_decode(smi)
         gid = assignments.graph_db_add_graph(gdb, smi, g)
 
-        gdb.graphs[gid] = g
-        gdb.smiles[gid] = smi
-        gdb.selections[topology.index_of(topology.atom)] = {
-            gid: {k: v for k, v in enumerate(graphs.graph_atoms(g))}
-        }
         gde = assignments.graph_db_entry()
```

Comment:
So I *think* this already happens here: https://github.com/trevorgokey/besmarts/blob/8251e4e4932f588f3184eb700f1c9767af48d71b/besmarts-core/python/besmarts/core/assignments.py#L1011-L1019

Reply:
Yes good eye; this is a case where I wrote the example, said "hey this is a common task that should go in core" but then never went back and refactored.